e2e-tests/cypress/tests/integration/channels/enterprise/system_console/channel_moderation/helpers.ts

```diff
@@ -31,7 +34,7 @@ export const disablePermission = (permission) => {
 };
 
 // # Saves channel config and navigates back to the channel config page if specified
-export const saveConfigForChannel = (channelName = false, clickConfirmationButton = false) => {
+export const saveConfigForChannel = (channelName: string = null, clickConfirmationButton = false) => {
```

Comment:
Was this always a `string`? Looks like it used to be a `boolean`, but we might have just been checking for a falsy case erroneously.

Reply:
The variable name suggests a string; additionally that variable is used as input for the type function; see line 59

```ts
if (channelName) {
    // # Search for the channel.
    cy.get('.DataGrid_searchBar').within(() => {
        cy.findByPlaceholderText('Search').type(`${channelName}{enter}`); // here
    });
    cy.findByText('Edit').click();
}
```